src/core/key-dictionary.ts

```diff
@@ -1,7 +1,9 @@
+
 const handleClipboardPaste = () => {
-  console.log("Cmd+Shift+V")
-}
+// tslint:disable-next-line:no-console
+  console.log("Cmd+Shift+V");
```

Comment:
you can remove all console log statements

Reply:
Done!